plugin-server/src/cdp/_tests/fixtures.ts

```diff
@@ -190,3 +191,31 @@ export const createExampleInvocation = (
         queuePriority: 0,
     }
 }
+
+export const createInvocationWithStatus = (
```

Comment:
Same here - hog functions dont get paused so I'm not sure what this is for. They already have an `enabled` property which is their equivalent